website/versioned_docs/version-0.61/debugging.md

```diff
@@ -161,6 +161,7 @@ On Android 5.0+ devices connected via USB, you can use the [`adb` command line t
 `adb reverse tcp:8081 tcp:8081`
 
 <!-- alex ignore host-hostess -->
+
```

Comment:
this and the other *.md changes are the result of running `yarn prettier`, can remove the change if wanted but other users will see this so we may want to keep it

Reply:
Going on and pulling this out since I found another title to fix anyway.